src/frontend/components/SelectContent/index.spec.tsx

```diff
@@ -577,6 +698,7 @@ describe('<SelectContent />', () => {
         </QueryClientProvider>,
       ),
     );
+    userEvent.click(screen.getByRole('tab', { name: /videos/i }));
```

Comment:
shouldn't you test the click perform some action ?

Reply:
It's the case: it allows to click on `Add a video` just after